src/containers/token-bridge/token-bridge-card.js

```diff
@@ -0,0 +1,291 @@
+              type="primary"
+              key="switch-chain"
+              onClick={async () => {
+                // Tries to automatically switch the chain...
+                try {
+                  await tokenBridgeApi.destination.switchChain();
+                } catch {
+                  // If the call fails, it means that it's not supported.
+                  // This happens for the native Ethereum Mainnet and well-known testnets,
+                  // such as Ropsten and Kovan. Apparently this is due to security reasons.
+                  message.error({
+                    key: "switch-network-fail",
+                    content: "Failed to switch network. You need to do that manually through MetaMask.",
+                  });
+                }
+              }}
+            >
+              Switch Network
+              <Icon type="arrow-right" />
+            </Button>
+          ) : null,
+        ]
+      : [
+          <Button key="skip" disabled={isSwapStepRunning} onClick={() => next({ wantedChainId: destinationChainId })}>
+            Skip
+            <Icon type="arrow-right" />
+          </Button>,
+        ];
+
+  const explainerProps = isSupportedSideChain(wantedChainId)
+    ? {
+        ethereumChainId: counterPartyChainIdMap[wantedChainId],
+        xDaiChainId: wantedChainId,
+      }
+    : isSupportedMainChain(wantedChainId)
+    ? {
+        ethereumChainId: wantedChainId,
+        xDaiChainId: counterPartyChainIdMap[wantedChainId],
+      }
+    : isSupportedSideChain(destinationChainId)
+    ? {
+        ethereumChainId: originChainId,
+        xDaiChainId: destinationChainId,
+      }
+    : {
+        ethereumChainId: destinationChainId,
+        xDaiChainId: originChainId,
+      };
+
+  const Explainer =
+    isSupportedSideChain(wantedChainId) || isSupportedSideChain(destinationChainId) ? ExplainerXDai : ExplainerEthereum;
+
+  return (
+    <StyledCard
+      title={
+        <>
+          Swap {originTokenSymbol}/{destinationTokenSymbol}
+        </>
+      }
+      actions={actions}
+    >
+      <Explainer {...explainerProps} />
+      <StyledDivider />
+      <SteppedContent
+        size="small"
+        current={step}
+        steps={[
+          {
+            title: (
+              <>
+                Convert {originTokenSymbol} to {destinationTokenSymbol}
+              </>
+            ),
+            children() {
+              return <SwapStep onRunning={runSwapStep} onDone={completeSwapStep} />;
+            },
+          },
+          {
+            title: `Switch to ${chainIdToNetworkName[wantedChainId ?? tokenBridgeApi.destination.chainId]}`,
+            children() {
+              return (
+                <SwitchNetworkMessage
+                  title={
+                    <>
+                      <span role="img" aria-label="tada">
+                        🎉
+                      </span>{" "}
+                      {originTokenSymbol} successfully converted!{" "}
```

Comment:
This message is displayed if even if the user clicks "Skip"

Reply:
Yeah, I thought about changing the message, but left as it was. Perhaps I can check if any transactions were submitted and display a different message.